packages/grafana-ui/src/components/Forms/InlineField.tsx

```diff
@@ -54,7 +54,7 @@ export const InlineField: FC<Props> = ({
     <div className={cx(styles.container, className)} {...htmlProps}>
       {labelElement}
       <div className={styles.childContainer}>
-        {React.cloneElement(children, { invalid, disabled, loading })}
+        <div className={styles.rowContainer}>{React.cloneElement(children, { invalid, disabled, loading })}</div>
```

Comment:
🤔 I am not sure what will be the side effects of wrapping the children into a div (this component is being used widely inside grafana). Also, while reviewing the PR I noticed we have another component called `InlineFieldRow`, did you try using that for the Annotations loading issue?

Reply:
Thanks for the comment!
So at the start I was also just using some kind of row to fix the Annotations issue, but then I realised that this seems to be something that happens in other places (like the screenshots I added) so I figured out that the layout seemed to have a regression from this change:

Before:
```
    <div className={cx(styles.container, className)} {...htmlProps}>
      {labelElement}
      {React.cloneElement(children, { invalid, disabled, loading })}
    </div>
```

After:
```
<div className={cx(styles.container, className)} {...htmlProps}>
      {labelElement}
      <div className={styles.childContainer}>
        {React.cloneElement(children, { invalid, disabled, loading })}
        {invalid && error && (
          <div className={cx(styles.fieldValidationWrapper)}>
            <FieldValidationMessage>{error}</FieldValidationMessage>
          </div>
        )}
      </div>
    </div>
```

That top `div` is like a ` flex-row`, so when you had something in the children of `InlineField` that was wrapped in Fragments, it would lay it all out horizontally, but after the change to introduce the error message underneath the children, now the children are rendered vertically. That is why I thought we should fix it more generally in the `InlineField`, although I definitely agree I'm skeptical of what repercussions this can have.
Do you think it's better to fix it case by case?

Reply:
It is safer, but I don't think is better 😓, maybe someone from @grafana/observability-squad  can help us to assess possible side effects? As this component is used in many Query Editors

Reply:
hi (i'm from @grafana/observability-logs-and-traces  , got pinged by the comment) i see that this has been answered in the linked slack discussion. if there's any other question, ping us again please 😄